common/src/main/java/com/revenuecat/purchases/common/ReceiptInfo.kt

```diff
@@ -10,8 +10,8 @@ class ReceiptInfo(
     val storeProduct: StoreProduct? = null,
 
     // TODO BC5 - replace price/currency with making a single PricingPhase for amazon observermode products?
-    val price: Double? = 1.0, // only passed for amazon observer mode
-    val currency: String? = "USD" // only passed for amazon observer mode
```

Comment:
This was making a test fail and I actually think changing it to null is a better idea. This only gets used when posting an observer mode purchase